spec/services/manifest_builder/license_builder_spec.rb

```diff
@@ -20,21 +20,5 @@
         expect(manifest["license"]).to eq "https://creativecommons.org/licenses/by-nc/4.0/"
       end
     end
```

Comment:
How are these changes related to the `prepopulate` change?

Reply:
I have no idea how this worked before! It seems like it was a bug. If there's a license field, why would it just be empty if you save a value?